xiblint/rules/named_colors.py

```diff
@@ -24,16 +28,26 @@ def check(self, context):  # type: (XibContext) -> None
             if element.parent.tag == 'namedColor':
                 continue
 
+            # Skip <color> tags part of a system color definition
+            if element.parent.tag == 'systemColor':
+                continue
+
             # Skip colors with alpha (if configured)
             if ignore_alpha and element.get('alpha') != '1':
```

Comment:
This seems to have had a bug before. Lets fix by checking if alpha is None
```suggestion
            if ignore_alpha and element.get('alpha') is not None and element.get('alpha') != '1':
```